src/main/java/carracing/domain/CarRacingProgram.java

```diff
@@ -0,0 +1,103 @@
+package carracing.domain;
+
+import carracing.view.CarInfoPrint;
+import carracing.view.CarRacingInfoPrint;
+
+import java.util.*;
+
+public class CarRacingProgram {
+    private List<Car> carList;
```

Comment:
아래 생성자에서 할당 할 필요 없이 이렇게 멤버 변수 선언할때 할당하는 부분이 더 좋을것 같아요.
그리고 생성자에서 할당한 부분은 빼버리고요
```suggestion
    private final List<Car> carList = new ArrayList<>();
```

물론 생성자에 따라 할당을 안해야 할때도 있다면 지금 처럼 하는게 맞아요
그걸 의도 한게 아니라면 다른 생성자를 만들때에 실수할 여지가 생길 수 있을 것 같아요